driver/bpf/fillers.h

```diff
@@ -2855,6 +2856,16 @@ FILLER(execve_family_flags, true)
 	/* Parameter 26: exe_file mtime (last modification time, epoch value in nanoseconds) (type: PT_ABSTIME) */
 	time = _READ(inode->i_mtime);
 	return bpf_val_to_ring_type(data, bpf_epoch_ns_from_time(time), PT_ABSTIME);
```

Comment:
we need to remove the `return` here
```suggestion
	ret = bpf_val_to_ring_type(data, bpf_epoch_ns_from_time(time), PT_ABSTIME);
	CHECK_RES(res);
```